agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4759,6 +4759,10 @@ bool slave_thread::handle_client_steering_request(Socket *sd, ieee1905_1::CmduMe
 {
     const auto mid = cmdu_rx.getMessageId();
     LOG(DEBUG) << "Received CLIENT_STEERING_REQUEST_MESSAGE , mid=" << std::hex << int(mid);
+    auto rx_buffer        = cmdu_rx.getMessageBuff();
+    uint16_t request_mode = (uint16_t)(rx_buffer[17]);
```

Comment:
I really don't like this... The TLV has been defined already, so why not do the proper addClass and get the request_mode?